intake_bluesky/core.py

```diff
@@ -15,10 +15,108 @@
 import numpy
 import warnings
 import xarray
+import heapq
+from functools import wraps
 
+def to_event_pages(get_event_cursor):
+    """
+    Decorator that changes get_event_cursor which yields events,
```

Comment:
The numpydoc standard requires a _one-line_ summary. Below that, after a blank line, you can elaborate in a paragraph if needed. Context: https://nsls-ii.github.io/scientific-python-cookiecutter/the-code-itself.html#a-simple-function-with-inline-documentation

Of course for `help(to_event_pages)` and `to_event_pages?` it doesn't matter, but the rendered HTML view by sphinx and the `autosummary` output don't work properly unless you keep to the standard.